main/store/migrations/index.js

```diff
@@ -407,7 +407,76 @@ const migrations = {
     })
 
     return initial
-  }
+  },
+  21: initial => {
+    // add sepolia network information
+    if (!initial.main.networks.ethereum[11155111]) {
+      initial.main.networks.ethereum[11155111] = {
+        id: 11155111,
+        type: 'ethereum',
+        layer: 'testnet',
+        symbol: 'ETH',
+        name: 'Sepolia',
+        explorer: 'https://sepolia.etherscan.io',
+        gas: {
+          price: {
+            selected: 'standard',
+            levels: { slow: '', standard: '', fast: '', asap: '', custom: '' }
+          }
+        },
+        connection: {
+          primary: { on: true, current: 'infura', status: 'loading', connected: false, type: '', network: '', custom: '' },
+          secondary: { on: false, current: 'custom', status: 'loading', connected: false, type: '', network: '', custom: '' }
+        },
+        on: false
+      }
+    }
+
+    if (!initial.main.networksMeta.ethereum[11155111]) {
+      initial.main.networksMeta.ethereum[11155111] = {
+        gas: {
+          fees: {},
+          price: {
+            selected: 'standard',
+            levels: { slow: '', standard: '', fast: '', asap: '', custom: '' }
+          }
+        }
+      }
+    }
+    
+    // we removed support for the following goerli RPCs so reset the connections 
+    // to defaults when the user was previously connecting to them
+    const removedGoerliRPCs = ['mudit', 'slockit', 'prylabs']
+    const goerliPrimaryConnection = initial.main.networks.ethereum[5].connection.primary.current
+    const goerliSecondaryConnection = initial.main.networks.ethereum[5].connection.secondary.current
+    
+    if (removedGoerliRPCs.includes(goerliPrimaryConnection)) {
+      initial.main.networks.ethereum[5] = {
```

Comment:
can probably clean up all the deeply nested references with something at the top like `const goerli = initial.main.networks.ethereum[5]`